File: week-1/day-2/functions/subint/tests/TestSubInt.cs
using Microsoft.ViusalStudio.TestTools.UnitTesting;
using SubInt;

namespace SubInt.Tests
{
	[TestClass()]
	public class TestSubInt
	{
		[TestMethod()]
		public void TestSubIntHappyCase()
		{
			Assert.AreEqual({0, 1, 4}, SubInt.SubInt(1, {1, 11, 34, 52, 61}));
		}

		[TestMethod()]
		public void TestSubIntSadCase()
		{
			Assert.AreEqual({}, SubInt.SubInt(9, {1, 11, 34, 52, 61}));
		}
	}
}

File: week-1/day-2/functions/subint/tests/test_subint.py
import unittest
from subint import subint


class Subint(unittest.TestCase):

    def test_subint_happy_case(self):
        self.assertEqual([0, 1, 4], subint(1, [1, 11, 34, 52, 61]))

    def test_subint_sad_case(self):
        self.assertEqual([], subint(9, [1, 11, 34, 52, 61]))

if __name__ == '__main__':
    unittest.main()
